src/info/repo.rs

```diff
@@ -337,23 +339,21 @@ fn bytes_to_human_readable(bytes: u128) -> String {
 }
 
 fn git_time_to_formatted_time(time: &Time, iso_time: bool) -> String {
-    let (offset, _) = match time.offset_minutes() {
-        n if n < 0 => (-n, '-'),
-        n => (n, '+'),
-    };
-
-    let offset = FixedOffset::west(offset);
-    let dt_with_tz = offset.timestamp(time.seconds(), 0);
     if iso_time {
-        dt_with_tz
-            .with_timezone(&chrono::Utc)
-            .to_rfc3339_opts(chrono::SecondsFormat::Secs, true)
+        to_rfc3339(HumanTime::from(time.seconds()))
     } else {
-        let ht = HumanTime::from(dt_with_tz);
+        let ht = HumanTime::from_duration_since_timestamp(time.seconds().unsigned_abs());
         format!("{}", ht)
     }
 }
 
+fn to_rfc3339<T>(dt: T) -> String
```

Comment:
It might be cool if we added some tests for these functions. Unfortunately we didn't have any prior tests for duration formatting to confirm consistent behavior.

Reply:
Yes, can do that

Reply:
@spenserblack You think those tests are sufficient?

Reply:
Yeah, those look good to me :+1:
Thanks!